counterpartylib/lib/blocks.py

```diff
@@ -1024,6 +1028,11 @@ def follow(db):
             block_index += 1
 
         else:
+            if not config.PARSE_MEMPOOL:
+                db.wal_checkpoint(mode=apsw.SQLITE_CHECKPOINT_PASSIVE)
+                time.sleep(config.BACKEND_POLL_INTERVAL)
```

Comment:
I put all this block in a big `if`. I don't think it's worth here to make a function for this 2 lines.